File: unity_app/Assets/Scripts/Core/Enums/VendorEnums.cs
namespace EventPlannerSim.Core
{
    /// <summary>
    /// Represents the category of vendor services.
    /// </summary>
    public enum VendorCategory
    {
        Caterer,
        Entertainer,
        Decorator,
        Photographer,
        Florist,
        Baker,
        RentalCompany,
        AVTechnician,
        Transportation,  // Post-MVP
        Security         // Post-MVP
    }

    /// <summary>
    /// Represents the quality/price tier of a vendor.
    /// </summary>
    public enum VendorTier
    {
        Budget,
        Standard,
        Premium,
        Luxury
    }
}

File: original-source/Models/Enums/VendorEnums.swift
import Foundation

enum VendorCategory: String, CaseIterable, Codable, Hashable {
    case caterer
    case entertainer
    case decorator
    case photographer
    case florist
    case baker
    case rentalCompany
    case avTechnician
    case transportation
    case security
}

enum VendorTier: String, CaseIterable, Codable, Hashable {
    case budget
    case standard
    case premium
    case luxury
}
